#include "mainwindow.h"
#include "ui_mainwindow.h"

//this file contains function that connects all buttons to keyboard.
//there's no much i can do against its largeness.

void MainWindow::connectAllButtons() {
    keyboard->connectButton(ui->pushButton_1);
    keyboard->connectButton(ui->pushButton_2);
    keyboard->connectButton(ui->pushButton_3);
    keyboard->connectButton(ui->pushButton_4);
    keyboard->connectButton(ui->pushButton_5);
    keyboard->connectButton(ui->pushButton_6);
    keyboard->connectButton(ui->pushButton_7);
    keyboard->connectButton(ui->pushButton_8);
    keyboard->connectButton(ui->pushButton_9);
    keyboard->connectButton(ui->pushButton_10);
    keyboard->connectButton(ui->pushButton_11);
    keyboard->connectButton(ui->pushButton_12);
    keyboard->connectButton(ui->pushButton_13);
    keyboard->connectButton(ui->pushButton_14);
    keyboard->connectButton(ui->pushButton_15);
    keyboard->connectButton(ui->pushButton_16);
    keyboard->connectButton(ui->pushButton_17);
    keyboard->connectButton(ui->pushButton_18);
    keyboard->connectButton(ui->pushButton_19);
    keyboard->connectButton(ui->pushButton_20);
    keyboard->connectButton(ui->pushButton_21);
    keyboard->connectButton(ui->pushButton_22);
    keyboard->connectButton(ui->pushButton_23);
    keyboard->connectButton(ui->pushButton_24);
    keyboard->connectButton(ui->pushButton_25);
    keyboard->connectButton(ui->pushButton_26);
    keyboard->connectButton(ui->pushButton_27);
    keyboard->connectButton(ui->pushButton_28);
    keyboard->connectButton(ui->pushButton_29);
    keyboard->connectButton(ui->pushButton_30);
    keyboard->connectButton(ui->pushButton_31);
    keyboard->connectButton(ui->pushButton_32);
    keyboard->connectButton(ui->pushButton_33);
    keyboard->connectButton(ui->pushButton_34);
    keyboard->connectButton(ui->pushButton_35);
    keyboard->connectButton(ui->pushButton_36);
    keyboard->connectButton(ui->pushButton_37);
    keyboard->connectButton(ui->pushButton_38);
    keyboard->connectButton(ui->pushButton_39);
    keyboard->connectButton(ui->pushButton_40);
    keyboard->connectButton(ui->pushButton_41);
    keyboard->connectButton(ui->pushButton_42);
    keyboard->connectButton(ui->pushButton_43);
    keyboard->connectButton(ui->pushButton_44);
    keyboard->connectButton(ui->pushButton_45);
    keyboard->connectButton(ui->pushButton_46);
    keyboard->connectButton(ui->pushButton_47);
    keyboard->connectButton(ui->pushButton_48);
    keyboard->connectButton(ui->pushButton_49);
    keyboard->connectButton(ui->pushButton_50);
    keyboard->connectButton(ui->pushButton_51);
    keyboard->connectButton(ui->pushButton_52);
    keyboard->connectButton(ui->pushButton_53);
    keyboard->connectButton(ui->pushButton_54);
    keyboard->connectButton(ui->pushButton_55);
    keyboard->connectButton(ui->pushButton_56);
    keyboard->connectButton(ui->pushButton_57);
    keyboard->connectButton(ui->pushButton_58);
    keyboard->connectButton(ui->pushButton_59);
    keyboard->connectButton(ui->pushButton_60);
    keyboard->connectButton(ui->pushButton_61);
    keyboard->connectButton(ui->pushButton_62);
    keyboard->connectButton(ui->pushButton_63);
    keyboard->connectButton(ui->pushButton_65);
    keyboard->connectButton(ui->pushButton_66);
    keyboard->connectButton(ui->pushButton_67);
    keyboard->connectButton(ui->pushButton_68);
    keyboard->connectButton(ui->pushButton_69);
    keyboard->connectButton(ui->pushButton_70);
    keyboard->connectButton(ui->pushButton_72);
}
